src/OpenApi/Factory/OpenApiFactory.php

```diff
@@ -843,4 +873,61 @@ private function mergeParameter(Parameter $actual, Parameter $defined): Paramete
 
         return $actual;
     }
+
+    /**
+     * @param string[]              $errors
+     * @param array<string, string> $responseMimeTypes
+     */
+    private function addOperationErrors(Operation $operation, array $errors, array $responseMimeTypes, ResourceMetadataCollection $resourceMetadataCollection, Schema $schema, \ArrayObject $schemas): Operation
+    {
+        $existingResponses = null;
+        foreach ($errors as $error) {
+            if (!is_a($error, ProblemExceptionInterface::class, true)) {
+                $this->logger?->warning(\sprintf('The error class "%s" does not implement "%s". Did you forget a use statement?', $error, ProblemExceptionInterface::class));
+            }
+
+            $status = null;
+            $description = null;
+            try {
+                /** @var ProblemExceptionInterface */
+                $exception = (new \ReflectionClass($error))->newInstanceWithoutConstructor();
```

Comment:
`new $error();` should do the trick. No need for reflection here.

Reply:
I disagree, it's possible that an exception in userland has constructor arguments

Reply:
But then constructing an instance without calling the constructor will likely create an object in an invalid sate, which is worse than an error right?

What if the title is passed as parameter in constructor for instance?